test/utils.test.js

```diff
@@ -0,0 +1,24 @@
+
+// IMPORT MODULES under test here:
+import { renderHair } from '../utils.js';
```

Comment:
great job with testing and renderFunction

Reply:
Thank you it was very challenging !